### DISCUSSION TITLE
Cloudlog callsign lookup

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#2034] [General]
I am finding that Cloudlog does not return proper contact locations when displaying them on custom maps in analytics, or in the logbook QSO info itself. What am I doing wrong? How would I properly set Cloudlog to check QRZ.com consistently?

Thanks & 73
Scott W1AL

**Comment by magicbug** [comment]
Sounds odd, so if grids are supplied that's used else we use the central point for the DXCC of the callsign

**Comment by soakland5** [comment]
I have the account setting to "yes" for grid lookup. Anything else I need to set?

**Reply by magicbug** [reply]
Nope thats everything.
